locator/tablets.hh

```diff
@@ -357,6 +357,9 @@ public:
     /// \throws std::logic_error If the given id does not belong to this instance.
     dht::token_range get_token_range(tablet_id id) const;
 
+    /// Returns the primary replica for the tablet
+    host_id get_primary_replica(tablet_id id) const;
+
```

Comment:
@asias @bhalevy we should use this for repair. Does it need to be integrated into effective_replication_map?

Reply:
> @asias @bhalevy we should use this for repair. Does it need to be integrated into effective_replication_map?

you mean to have such a function for vnodes?
i don't think it's necesaary since tablet repair has it's own path that can diverge from vnodes.
Also, for repair, we also need a variant of this function (or an optional parameter to it, to select a primary replica within a given dc)